meetings/2020-03-03.md

```diff
@@ -0,0 +1,178 @@
+
+Michael: If we have good candidates in (eg.) the moderation team, we’d probably want to reach out to them. For example: we had someone who was fairly active from the AMP team, we should probably invite them as well.
+
+Eemeli: Do we have a clear listing of who’s on the bootstrapped CoC panel anywhere?
+
+Michael: Not yet, I think there’s an issue but we haven’t landed on a list yet.
+
+Joe: I’ve got this on my list, I’ll try to help push it forward as well.
+
+* Add Foundation-wide copyright guidance [#414](https://github.com/openjs-foundation/cross-project-council/pull/414)
+
+Joe: This is on Tobie’s plate. He’s sick atm. I’m going to reach out about this and #386, and see if I can help him get these over the finish line. Nothing new on these two, I’ll try to help get them over the finish line.
+
+* Add post-graduation checklist [#386](https://github.com/openjs-foundation/cross-project-council/pull/386)
+
+* Responsible security disclosures [#326](https://github.com/openjs-foundation/cross-project-council/issues/326)
+
+Michael: reached out to Marcin, nothing to report this week but he is going to try to attend next week to give an update.
+
+* Add admin policy docs [#185](https://github.com/openjs-foundation/cross-project-council/issues/185)
+
+Joe: I have an open PR, will adjust based on comments.
+
+Ben: Already on it, should have the updates committed sometime today.
+
+* Project Directed Funding [#90](https://github.com/openjs-foundation/cross-project-council/issues/90)
+
+Michael: Board discussed this week, believe we should have some text shared within next
+     week or so on the aspect of Companies donating to specific projects.
+
+OpenJS Collaboration Network [#474](https://github.com/openjs-foundation/cross-project-council/issues/474)
+
+Michael: Just like you can boot up a project within the ecosystem, we’re interested in offering a bootstrap process with similar resources the CPC provides for inter-project collaboration. Probably going to be based on subject: eg. Security, diagnostics, etc. I’m just proposing this because the WGs are already organic for people who’re already in the projects that host them, but it may not be easy for outside groups to have a way in to potentially collaborate. Is this a good thing, or bad thing? Is this just something that should be left for the working groups?
+
+Darcy: I think this is a good thing. I’d like to help formalize this a little more.
+
+Joe: I like this idea too. My only concern is around how it would work and the overlap with what node is already doing, is it elevating work from over groups over node?
+
+Michael: I don’t think we should push groups to move up just because we know them. Part of the underlying explanation of this could be for groups to get more involved in the node project. Maybe for that reasoning this should just stay a part of the node project. We’re just interested in bringing projects together in a way that they haven’t been able to before.
+
```

Comment:
```suggestion
Michael: I don’t think we should push groups to move up just because we know them. Part of the underlying explanation of this could be for groups to get more involved in the node project. Maybe for that reasoning this should just stay a part of the node project. We’re just interested in bringing people to collaborate in a way that they haven’t been able to before.
```